try{
	const.resultado = dividir(10, 0);
	console.log(resultado);
} catch (error){
console.log('Error:' error.message);
} finally {
	console.log('Operacion finalizada')
}
function dividir(a, b) {
	if (b===0) {
		
	}
}